src/module.c

```diff
@@ -4914,6 +4953,9 @@ void moduleBlockedClientTimedOut(client *c) {
     ctx.blocked_privdata = bc->privdata;
     bc->timeout_callback(&ctx,(void**)c->argv,c->argc);
     moduleFreeContext(&ctx);
+    if (!bc->blocked_on_keys) {
```

Comment:
@madolson added the background time tracking on timeout as well for modules (tested on test named "check blocked command that uses RedisModule_MeasureTimeStart() is tracking background time even in timeout" ). 
With regards replyToBlockedClientTimedOut on lists, sorted sets or streams, given that timeout occurs on keys there is no reason to track that time correct?